### DISCUSSION TITLE
[LINUX/BASH] What are the equivalent of "$ export CFLAGS+=" -I/path/"" and "$ export CXXFLAGS+=" -I/path/"" inside the makefile?

### DISCUSSION THREAD

**Discussion by cinnamonmatexfce** [mamedev/discussions#76] [Q&A]
What `USER-CONFIGURABLE OPTIONS` should I use?
`ARCHOPTS=`
or
`ARCHOPTS_C=`
or
`ARCHOPTS_CXX=`
or
`OVERRIDE_CC=`
or
`OVERRIDE_CXX=`

Looking at the syntax, for what I understand, I think I'd go for `ARCHOPTS_C=` and `ARCHOPTS_CXX`.
Right?

Thanks

**Comment by rb6502** [comment]
What are you actually trying to do?  MAME should compile fine as-is on pretty near any Linux distro.
